### PR TITLE
Fix broken support for loading file variables

### PR DESCRIPTION
**Pull request by datakurre**
When has file loading of file variable been tested to really work for the last time?

We found two issues:

* Refactoring for "keycloak auth interceptor" added assumption for all responses to be JSON. This is not true for https://docs.camunda.org/manual/latest/reference/rest/execution/local-variables/get-local-variable-binary/, which returns raw binary. https://github.com/camunda/camunda-external-task-client-js/commit/30c04c2ea87cd1d8aac30dbe86882b0b33cf0462#diff-3b4d04702c2e044519a899f8352422660b7eeeab287be393b1796e6c0bccaecdR49

* Even older code assumed https://docs.camunda.org/manual/latest/reference/rest/execution/local-variables/get-local-variable-binary/ to return base64 encoded data.

This pull has currently two commits. The first one adds saving and loading a file variable into integration test. That only should be able to reproduce the issue (and reveal the wrong assumptions in unit tests). The second commit tries to fix this.

### CODE REVIEW

File: lib/Client.test.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ describe("Client", () => {
         jest.advanceTimersByTime(0.5 * customClientOptions.interval);
 
         expect(pollSpy).toHaveBeenCalledTimes(3);
-        expect(engineService.fetchAndLock).toHaveBeenCalledTimes(2);
+        expect(engineService.fetchAndLock).toHaveBeenCalledTimes(1);
```
Comment (datakurre) [inline lib/Client.test.js:188]: I have no idea, why this change was needed. Actually, on my local environment, with Node  v14.15.4 even more counters were wrong. I assume reason being removal of  `json()` from got mock.

**Comment by mboskamp** [comment]
Hi @datakurre, 
thank you for your contribution! 
We will take a look at it and come back to you as soon as possible.
Cheers,
Miklas

**Comment by tasso94** [comment]
Hi @datakurre,

Thank you for your great contribution. 🚀 
I've merged it into the codebase.

Best,
Tassilo
